docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+# docker-compose for socorro-collector that imitates the Mozilla
```

Comment:
Is there a way to do this where variables aren't duplicated between the two containers?
You can set things in a docker-compose.yml with ${VAR} if they are env variables, and you can set env variables with ENV.